bin/peanut.dart

```diff
@@ -31,20 +53,52 @@ main(List<String> args) async {
   var dir = result[_directoryFlag] as String;
   var branch = result['branch'] as String;
 
-  var mode = result['mode'] as String;
+  var buildTool = result['build-tool'] as String;
+
+  String pubBuildMode;
+  String buildRunnerConfig;
+  if (buildTool == 'build') {
```

Comment:
Unless you think there are lots of people that are going to get stuck you can consider a breaking release update that switches to build_runner instead of a transition supporting either. Folks using pub build can stick with old releases and folks using build_runner can upgrade. Should remove complexity in the mean time.

Reply:
Yeah – maybe later. I'll do a follow-up CL to change the default to `build` if `build.release.yaml` exists.